modules/thunder_search/config/optional/search_api.index.content.yml

```diff
@@ -0,0 +1,316 @@
+    type: boolean
+    dependencies:
+      module:
+      - node
+  rendered_item:
+    label: 'Rendered HTML output'
+    property_path: rendered_item
+    type: text
+    configuration:
+      roles:
+        anonymous: anonymous
+      view_mode:
+        'entity:media':
+          gallery: default
+          image: default
+          instagram: default
+          pinterest: default
+          twitter: default
+          video: default
+        'entity:node':
+          article: default
+          page: default
+        'entity:taxonomy_term':
+          channel: default
+          tags: default
+  status:
+    label: status
+    datasource_id: 'entity:node'
+    property_path: status
+    type: boolean
+    indexed_locked: true
+    type_locked: true
+    dependencies:
+      module:
+      - node
+  sticky:
+    label: 'Sticky at top of lists'
+    datasource_id: 'entity:node'
+    property_path: sticky
+    type: boolean
+    dependencies:
+      module:
+      - node
+  term_changed:
+    label: Changed
+    datasource_id: 'entity:taxonomy_term'
+    property_path: changed
+    type: date
+    dependencies:
+      module:
+      - taxonomy
+  term_status:
+    label: Published
+    datasource_id: 'entity:taxonomy_term'
+    property_path: status
+    type: boolean
+    dependencies:
+      module:
+      - taxonomy
+  type:
+    label: Type
+    property_path: aggregated_field
+    type: string
+    configuration:
+      type: union
+      fields:
+      - 'entity:media/bundle'
+      - 'entity:node/type'
+      - 'entity:taxonomy_term/vid'
+  uid:
+    label: uid
+    datasource_id: 'entity:node'
+    property_path: uid
+    type: integer
+    indexed_locked: true
+    type_locked: true
+    dependencies:
+      module:
+      - node
+datasource_settings:
+  'entity:media':
+    bundles:
+      default: true
+      selected: {  }
+    languages:
+      default: true
+      selected: {  }
+  'entity:node':
+    bundles:
+      default: true
+      selected: {  }
+    languages:
+      default: true
+      selected: {  }
+  'entity:taxonomy_term':
+    bundles:
+      default: true
+      selected: {  }
+    languages:
+      default: true
+      selected: {  }
+processor_settings:
+  add_url: {  }
+  aggregated_field: {  }
+  content_access:
+    weights:
+      preprocess_query: -30
+  hierarchy:
+    fields:
+      field_channel: taxonomy_term-parent
+    weights:
+      preprocess_index: -45
+  html_filter:
+    all_fields: true
+    fields:
+    - label
+    - media_author
+    - name
+    - rendered_item
+    - type
+    title: true
+    alt: true
+    tags:
+      b: 2
+      h1: 5
+      h2: 3
+      h3: 2
+      strong: 2
+    weights:
+      preprocess_index: -15
+      preprocess_query: -15
+  ignorecase:
+    all_fields: true
+    fields:
+    - label
+    - media_author
+    - name
+    - rendered_item
+    - type
+    weights:
+      preprocess_index: -20
+      preprocess_query: -20
+  rendered_item: {  }
+  stopwords:
```

Comment:
Stopwords should be handled by a real server, not by the search api => disable it